components/admin_console/admin_definition.jsx

```diff
@@ -6271,6 +6271,17 @@ const AdminDefinition = {
                         isHidden: it.not(it.licensedForFeature('SharedChannels')),
                         isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.EXPERIMENTAL.FEATURES)),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ExperimentalSettings.EnableAppBar',
+                        label: t('admin.experimental.enableAppBar.title'),
+                        label_default: 'Enable App Bar:',
+                        help_text: t('admin.experimental.enableAppBar.desc'),
+                        help_text_default: 'When true, the experimental App Bar is enabled. Any remaining channel header buttons from plugins that do not explicitly register an App Bar button will be automatically moved to the App Bar, which may cause rendering issues.',
```

Comment:
```suggestion
                        help_text_default: 'When true, all integrations move from the channel header to the App Bar. Channel header plugin icons that haven't explicitly registered an App Bar icon will be moved to the App Bar which may result in rendering issues. See the documentation to learn more.',
```

Reply:
The documentation URL for this will be: https://docs.mattermost.com/welcome/what-changed-in-v70.html

Reply:
Thank you! Done in https://github.com/mattermost/mattermost-webapp/pull/10318/commits/96edd9244da664d194e56b86dffacb8b904fdd21